registry.go

```diff
@@ -0,0 +1,220 @@
+package gitlab
+
+import (
+	"fmt"
+	"net/url"
+	"time"
+)
+
+// RegistryService handles communication with the container registry related methods
```

Comment:
Just as a FYI... I try to keep comments below 80chars and LoC below 100chars. So this comment would then become:

```
// RegistryService handles communication with the container registry related
// methods of the Gitlab API.
```

A valid exception for comments is when a URL is longer the 80chars and is already on its own line and a valid exception for LoC is function signatures, as I always let them be on a single line. Thanks!